src/libcollections/btree/map.rs

```diff
@@ -1898,6 +1898,12 @@ impl<'a, K: Ord, V> VacantEntry<'a, K, V> {
         &self.key
     }
 
+    /// Take ownership of the key.
+    #[unstable(feature = "map_entry_keys", issue = "32281")]
```

Comment:
Could these point to a new issue? Pointing to a close issue  may unfortunately mean that these never come up for stabilization :(